pages/app.js

```diff
@@ -147,9 +145,9 @@ class App extends Component {
     if (this.state.totalGames < 5) {
       return (
         <div>
-          <button className='btn-scissors' onClick={() => { this.gameHandler('scissors') }}>SCISSORS</button>
-          <button className='btn-paper' onClick={() => { this.gameHandler('paper') }}>PAPER</button>
-          <button className='btn-rock' onClick={() => { this.gameHandler('rock') }}>ROCK</button>
+          <button data-behavior='btn-scissors' onClick={() => { this.gameHandler('scissors') }}>SCISSORS</button>
```

Comment:
You'll need to keep the classnames for styling purposes, but as you don't have any styling at the moment, it won't be an issue.